tests/pytests/test_info_modules.py

```diff
@@ -974,10 +988,7 @@ def test_syntax_errors_cluster(self):
     # Test counter on each shard
     for shardId in range(1, self.env.shardsCount + 1):
       shard_conn = self.env.getConnection(shardId)
-      info_dict = info_modules_to_dict(shard_conn)
-      syntax_error_count = info_dict[WARN_ERR_SECTION][SYNTAX_ERROR_SHARD_METRIC]
-      self.env.assertEqual(syntax_error_count, '2',
-                           message=f"Shard {shardId} has wrong syntax error count")
+      wait_for_info_metric(shard_conn, [WARN_ERR_SECTION, SYNTAX_ERROR_SHARD_METRIC], '2', msg=f"Shard {shardId} has wrong syntax error count")
```

Comment:
so the first shard that gets timeout hault the test and we lose the other shards information.